### PR TITLE
Corrections and additions

### PR DESCRIPTION
**Pull request by Variable-ind**
fix a typo and added duplication property to the selection docs

### CODE REVIEW

File: docs/user_manual/selecting.md [hunk 1]
```diff
@@ -40,6 +40,13 @@ If you want to delete the entire selection, the easiest way is to deselect every
 ### Inverting Selection
 Inverting the selection means that all of the currently selected pixels will get deselected, while all of the unselected pixels will get selected. You can do this by either pressing <kbd>Control + I</kbd>, or going to the Select menu and pressing **Invert**. If you have everything selected, inverting will clear the selection. Likewise, if you have nothing selecting, inverting will select everything.
 
+## Duplicating
+If you **don't** have an active selection, holding <kbd>Control + Alt + Mouse click</kbd> will *duplicate* whatever was below it and place it in active selection.
```
Comment (OverloadedOrama) [inline docs/user_manual/selecting.md:44]: Control + Alt + Mouse will not have any effect if you don't have an active _selection_. I think you probably meant to write [active transformation](https://orama-interactive.github.io/Pixelorama-Docs/user_manual/transforming#having-an-active-transformation).

File: docs/user_manual/selecting.md [hunk 2]
```diff
@@ -40,6 +40,13 @@ If you want to delete the entire selection, the easiest way is to deselect every
...
+
+:::tip Duplicating Quickly!
+**During active selection** <kbd>Control + Alt + Mouse click</kbd> will not only apply selection but will also **retain** it's contents, which you can use again. Handy when duplicating something multiple times.
```
Comment (OverloadedOrama) [inline docs/user_manual/selecting.md:47]: Same as above, I recommend changing "active selection" to "active transformation" and "apply selection" to "[apply transformation](https://orama-interactive.github.io/Pixelorama-Docs/user_manual/transforming#applying-a-transformation)", as well as "it's contents" to "its content".

**Review by OverloadedOrama — CHANGES_REQUESTED** [review]

File: docs/user_manual/selecting.md (hunk 2 above)
Comment (Variable-ind) [reply docs/user_manual/selecting.md]: done!

**Review by OverloadedOrama — APPROVED** [review]
Thank you!

**Pull request merged**